lib/fake_braspag/sales.rb

```diff
@@ -2,7 +2,9 @@ module FakeBraspag
   class Sales < Sinatra::Base
```

Comment:
If you like, some conditionals can be avoided below by creating a Hash such as:

```ruby
METHOD_SPECIFIC = {
  'CreditCard' => {
    views: {
      find: : get_credit_card_sale,
      authorize: :sales_authorize
    },
    input: {
      'paymentMethod' => 'CreditCard',
      'cardNumber' => @params['Payment']['CreditCard']['CardNumber'],
      'saveCard' => @params['Payment']['CreditCard']['SaveCard']
    }
  }
  'Boleto' = {
    views: {
      find: :get_boleto_sale
      authorize: sales_boleto
    },
    input: {
      'paymentMethod' => 'Boleto'
    }
  }
}
```

And then applying the modifications specified in the comments below.